.gitignore

```diff
@@ -76,11 +24,8 @@ pom.xml
 bazel*
 user.bazelrc
 
-# Atom conf
-.clang_complete
-
 # Database
 ciri/ciri.db*
 
-# Visual Studio Code conf
+# ignore all VisualStudioCode user settings
 .vscode/
```

Comment:
What is the reason some are removed and some aren't ?

Reply:
The global .gitignore genarated from `www.gitignore.io` will commit user settings, but I don't think we should keep user those settings. 

```
### VisualStudioCode ###
.vscode/*
!.vscode/settings.json
!.vscode/tasks.json
!.vscode/launch.json
!.vscode/extensions.json
```